content/modeling/modeling_1.md

```diff
@@ -0,0 +1,95 @@
+
+- The `core_web_vital_results_query()` macro is used to let the user classify the thresholds to be applied for the measurements. It returns the sql to provide the logic for the evaluation based on user defined thresholds (expected in the form of case when statements). Please make sure you set the results you would like the measurements to pass to `good` or align it with the `macro_core_web_vital_pass_query()` macro. ([source](https://github.com/snowplow/dbt-snowplow-web/blob/main/macros/core_web_vital_results_query.sql))
+
+- The `core_web_vital_pass_query()` macro is used to let the user define what counts as the overall pass condition for the core web vital measurements. ([source](https://github.com/snowplow/dbt-snowplow-web/blob/main/macros/core_web_vital_pass_query.sql))
+
+#### **Step 4:** Run the package
+
+If this is your first time processing the snowplow_web package then you can run the package the recommended way either through your CLI or from within dbt Cloud with the following command:
+
+```
+dbt run --selector snowplow_web
+```
+
+If you already have a working snowplow_web package and would like to enable the core web vitals module you do not need to rebuild the whole model from scratch. For the least amount of reprocessing impact execute the first run the following way:
+
+```
+dbt run -m snowplow_web.base --vars 'snowplow__start_date: <date_when_core_web_vital_tracking_starts>'
+```
+This way only the base module is reprocessed. The web model's update logic will recognize the newly enabled models and backfilling should start between the date you defined within snowplow_start_date and the upper limit defined by the variable `snowplow_backfill_limit_days` that is set for the web model.
+
+`Snowplow: New Snowplow incremental model. Backfilling`
+
+You can overwrite this limit for the backfilling process temporarily if needed:
+
+```
+# dbt_project.yml
+
+vars:
+  snowplow_web:
+    snowplow__backfill_limit_days: 1
+```
+
+After this you should be able to see all core web vitals models created. Any subsequent run from this point onwards could be carried out using the recommended web model running method - using the snowplow_web selector.
+
+```
+dbt run --selector snowplow_web
+```
+
+As soon as backfilling finishes, running the model results in both the web and the core web vital models being updated during the same run for the same period, both using the same latest set of data from the `_base_events_this_run` table. Please note that while the backfilling process lasts, no new web events are going to be processed.
+
+The following models will be generated:
+
+- **snowplow_web_vitals**: Incremental table used as a base for storing core web vital events (first event per pageview).
```

Comment:
Would be nice to either describe what is in these models or link out to the dbt docs that describes these columns.